addons/packages/pinniped/0.12.1/bundle/config/overlay/dex-deployment.yaml

```diff
@@ -29,7 +29,8 @@ spec:
         revision: "1"
     spec:
       containers:
-      - image: projects.registry.vmware.com/tce/dex:v2.30.2_vmware.1
+      #! we are currently manually building and pushing a dex image based on OSS upstream dex
+      - image: projects.registry.vmware.com/tce/dex:v2.31.0_vmware.1
```

Comment:
We discussed if this should be an upstream dex rather than our own built version at this stage in TCE.  @ankeesler